### DISCUSSION TITLE
Nuxt 3 Js  not work with Auth + Axios

### DISCUSSION THREAD

**Discussion by KhuloodBatis** [vercel/next.js#68348] [Help] [answered]
### Summary

Hello there ,

i was created Nuxt.js as frontend to work with Laravel BacKend 

i need use sanctum auth Laravel API

my issue when use Auth and Axios in Nuxtjs and Put it in nuxt.config.js file  
` npm run dev `

`that was Happend `

<img width="960" alt="Screenshot 2024-07-31 at 10 32 10 AM" src="https://github.com/user-attachments/assets/c9a59e6b-fc07-4b7f-b7dc-86ae4a608271">

there my Config 

`
export default {
   compatibilityDate: '2024-07-29',
  // Global page headers: https://go.nuxtjs.dev/config-head
  head: {
    title: 'jobs-frontend',
    htmlAttrs: {
      lang: 'en'
    },
    meta: [
      { charset: 'utf-8' },
      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
      { hid: 'description', name: 'description', content: '' },
      { name: 'format-detection', content: 'telephone=no' }
    ],
    link: [
      { rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }
    ]
  },

  // Global CSS: https://go.nuxtjs.dev/config-css
  css: [
  ],

  // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
  plugins: [
  ],

    devtools: { enabled: true },
    css: ['~/assets/css/main.css'],
    postcss: {
      plugins: {
        tailwindcss: {},
        autoprefixer: {},
      },
    },

  // Auto import components: https://go.nuxtjs.dev/config-components
  components: true,

  // Modules for dev and build (recommended): https://go.nuxtjs.dev/config-modules
  buildModules: [
    // https://go.nuxtjs.dev/tailwindcss
    '@nuxtjs/tailwindcss',
  ],

  // Modules: https://go.nuxtjs.dev/config-modules
  modules: [
    '@nuxtjs/axios',
    '@nuxtjs/auth',
  ],

  auth: {
    strategies: {
      local: {
        endpoints: {
          login: { url: '/api/login', method: 'post', propertyName: false },
          logout: { url: '/api/logout', method: 'post' },
          user: { url: '/api/user', method: 'get', propertyName: false }
        },
        tokenRequired: false,
        tokenType: false,
      }
    },
    localStorage : false,
  },

  axios: {
    baseURL: 'http://jobs.test', // Replace with your API base URL
    credentials: true,
  },
  // Build Configuration: https://go.nuxtjs.dev/config-build
  build: {
  }
}

`
Nuxt3 , packege.json 

`{
  "name": "jobs-frontend",
  "version": "1.0.0",
  "private": true,
  "scripts": {
    "dev": "nuxt dev",
    "dev-host": "NUXT_HOST=jobs.test NUXT_PORT=3333 nuxt dev",
    "build": "nuxt build",
    "start": "nuxt start",
    "generate": "nuxt generate"
  },
  "dependencies": {
    "@nuxtjs/axios": "^5.13.6",
    "axios": "^1.7.2",
    "bootstrap": "^5.3.3",
    "consola": "^3.2.3",
    "core-js": "^3.25.3",
    "cross-env": "^7.0.3",
    "defu": "^6.1.4",
    "last": "^1.1.0",
    "nuxt": "^2.15.8",
    "ufo": "^1.5.4",
    "vue": "^3.4.34",
    "vue-server-renderer": "^2.2.6",
    "vue-template-compiler": "^0.1.0"
  },
  "devDependencies": {
    "@nuxtjs/tailwindcss": "^5.3.3",
    "autoprefixer": "^10.4.19",
    "nuxt": "npm:nuxt3@latest",
    "postcss": "^8.4.40",
    "tailwindcss": "^3.4.7"
  }
}
`
i try to use packages but still didn't work with me 

### Additional information

```js
Nuxt3 , Auth , Axios , Laravel , sanctum , API
```

**Comment by icyJoseph** [comment, marked as answer]
I think you can go through this issue report, and try some of the stuff there, https://github.com/nuxt-community/axios-module/issues/536, looks like the issue has been going on for a while?

That being said, this is a Next.js discussions board. Not Nuxt :)
